perception/lidar_centerpoint_tvm/include/lidar_centerpoint_tvm/centerpoint_tvm.hpp

```diff
@@ -55,6 +55,22 @@ struct MixedInputs
   std::vector<int32_t> coords;
 };
 
+class LIDAR_CENTERPOINT_TVM_LOCAL TVMScatterIE : public tvm_utility::pipeline::InferenceEngine
```

Comment:
What do you think of having this class in `tvm_utility` instead? I don't see anything specific to scatter and it could be reused by other packages. Could be renamed to refer to tvm modules generally.

Reply:
Ah nevermind, it doesn't make sense because of `coords`.